bot/persistence/database_connector.py

```diff
@@ -57,6 +58,100 @@ def remove_module_role(self, role_id: int):
             db_manager.execute(queries.REMOVE_MODULE_ROLE, (role_id,))
             db_manager.commit()
 
+    def add_suggestion(self, author_id: int, timestamp: datetime.datetime) -> int:
+        """Adds a suggestion to the table "Suggestion".
+
+        Args:
+            author_id (int): The id of the user who submitted the suggestion.
+            timestamp (datetime.datetime): A timestamp when this suggestion has been submitted.
+
+        Returns:
+            int: The row id of the new entry.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            row_id = db_manager.execute(queries.INSERT_SUGGESTION, (author_id, timestamp)).lastrowid
```

Comment:
Is lastrowid the row number or actually the value in the id column. 

Because any vacuuming or similar things in the db will change the row numbers.

Reply:
> If a table contains a column of type INTEGER PRIMARY KEY, then that column becomes an alias for the ROWID. You can then access the ROWID using any of four different names, the original three names described above or the name given to the INTEGER PRIMARY KEY column. **All these names are aliases for one another and work equally well in any context.**

https://sqlite.org/autoinc.html